ClientApp/app/tabs/_layout.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import FontAwesome from '@expo/vector-icons/FontAwesome';
+import { Tabs } from 'expo-router';
+import { useRouter } from 'expo-router';
+
+export default function TabLayout() {
+  const router = useRouter();
+
+  return (
+    <Tabs
```

Comment:
can you make the spacing between the icons and their respective texts smaller?

Reply:
sure